doc/bn.tex

```diff
@@ -875,46 +880,31 @@ \subsection{Long Constants}
 
 \subsection{Long Constants - platform dependant}
 
-\index{mp\_set\_long}
+\index{mp\_set\_ulong}
 \begin{alltt}
-int mp_set_long (mp_int * a, unsigned long b);
+void mp_set_slong (mp_int * a, long b);
+void mp_set_ulong (mp_int * a, unsigned long b);
 \end{alltt}
 
 This will assign the value of the platform-dependent sized variable $b$ to the mp\_int $a$.
 
-To get the ``unsigned long'' copy of an mp\_int the following function can be used.
-
-\index{mp\_get\_long}
-\begin{alltt}
-unsigned long mp_get_long (mp_int * a);
-\end{alltt}
-
-This will return the least significant bits of the mp\_int $a$ that fit into an ``unsigned long''.
-
-\subsection{Long Long Constants}
-
-\index{mp\_set\_long\_long}
-\begin{alltt}
-int mp_set_long_long (mp_int * a, unsigned long long b);
-\end{alltt}
-
-This will assign the value of the 64-bit variable $b$ to the mp\_int $a$.
-
-To get the ``unsigned long long'' copy of an mp\_int the following function can be used.
+To retrieve the value, the following function can be used.
 
-\index{mp\_get\_long\_long}
+\index{mp\_get\_ulong}
 \begin{alltt}
-unsigned long long mp_get_long_long (mp_int * a);
+long mp_get_slong (mp_int * a);
+unsigned long mp_get_ulong (mp_int * a);
 \end{alltt}
 
-This will return the 64 least significant bits of the mp\_int $a$.
+This will return the least significant bits of the mp\_int $a$ that fit into an ``long''.
```

Comment:
You might insist on using "into" instead of "in" because you "move the bits into a bucket named 'long'" but it's still "a long", no vowel or mute H at the start of the word "long".

Are there really no native speakers left in this group?

Reply:
Sorry, I can do better than that. I removed the unsigned but didn't remove the "an".

It is just that I didn't took much care and I am not a big fan of using TeX for API docs. Generally I am a big TeX fan but not for this.

Reply:
fixed